testing/cross_browser_test.md

```diff
@@ -0,0 +1,56 @@
+## Cross-browser Testing
+
+Testing applications across multiple browsers has become necessary given the differing behaviors and support for different HTML, CSS, and JavaScript features with the majority of browsers. Tools like [Selenium](http://www.seleniumhq.org/) have arisen to allow an ease of testing multiple browsers using drivers, but even this has some limitations and added complexity.
+
+SauceLabs and BrowserStack were created to help ease some of the limitations and complexity with testing applications cross-browser with Selenium by providing a sort of "Selenium Server as a Service" through automation services.
```

Comment:
Might be worth calling out the benefit of using a service rather than rolling your own Selenium Grid deployment